### DISCUSSION TITLE
Identity Provider Integration

### DISCUSSION THREAD

**Discussion by MarkusGnigler** [keygen-sh/community#109] [Q&A]
Hy,

Is it planned or even better, already possible to integrate keygen api with some external identity providers such as keycloack?

Neat product BTW, great job :+1:

Greets 
Markus

**Comment by ezekg** [comment]
We're working on it via https://github.com/keygen-sh/keygen-api/issues/409. No definite timeline yet.

**Comment by MarkusGnigler** [comment]
Great, thanks for the hint.
